### ISSUE TITLE
FO4Edit: Hardcoded Update

### ISSUE THREAD

**Issue by fireundubh** [TES5Edit/TES5Edit#267]
SMEN records have this PNAM:

`[0000005B] <Warning: Could not be resolved, but is possibly hardcoded in the engine>`

With the console, you can arbitrarily add forms to formlists, dump those formlists, and clear them, so I created a plugin with an empty formlist, added this Form ID to the list, and dumped the list.

1. `AddFormToFormList 1000800 5B`
2. `DumpFormList 1000800`
3. `RevertFormList 1000800`

Unfortunately, dumping the formlist displays only the signature and Form ID, so all I could find out is that 0x5B has a SMBN signature. SMBN records don't show up in the console, so I couldn't look up the name.

tl;dr: SMEN PNAMs are SMBNs, specifically `[SMBN:0000005B]`.

This needs to be added to the hardcoded reference plugin.

I don't know if there are any other possibly hardcoded references. Let me know?

**Comment by zilav** [comment]
I don't know. Hardcoded references are usually discovered this way - you check existing records in CK without loading the game master file. So lets just wait for CK.

**Comment by fireundubh** [comment]
As long as this thread is still open...

`[0000034F]` is called Fatigue. We can add that AVIF record to the FO4 .dat file.
